### PR TITLE
Modify kitn storing, add kitn counting to the user and read kitn counts in top scores instead of counting reports

### PR DESCRIPTION
**Pull request by eugene-eko2000**

### CODE REVIEW

File: db/init.sql [hunk 1]
```diff
@@ -13,6 +13,10 @@ CREATE TABLE IF NOT EXISTS users (
   privacy varchar(255),
   agree_toc varchar(255),
   referral VARCHAR(32),
+  kitns_daily INT DEFAULT 0,
```
Comment (eldarm) [inline db/init.sql:16]: Are you sure we need so many fields? We did not discussed the design for that.

**Review by eldarm — CHANGES_REQUESTED** [review]

File: be/db.go [hunk 2]
```diff
@@ -23,24 +24,6 @@ func mysqlAddress() string {
 	return db
 }
 
-func validateResult(r sql.Result, e error, checkRowsAffected bool) error {
```
[the remark this answers is not in the document]
Comment (eugene-eko2000) [reply be/db.go:26]: Sure, I moved logging back.

File: db/init.sql (hunk 1 above)
Comment (eugene-eko2000) [reply db/init.sql:16]: We discussed it earlier with Robin. If you take a look at the KITN screen you'll see four numbers.
Simulator Screenshot - iPhone 15 Pro - 2023-12-22 at 16 05 22. We might also change that in the future.
So we have to store a detailed info about each kind of KITN count.

Comment (eldarm) [reply db/init.sql:16]: Ok,  think this design is too heavy, but ok for now.

**Review by eldarm — APPROVED** [review]
LGTM

**Pull request merged**